docs/content/ChoiceInputField.mdx

```diff
@@ -0,0 +1,155 @@
+---
+title: ChoiceInputField
+status: Alpha
+source: https://github.com/primer/react/blob/main/src/ChoiceInputField.tsx
+storybook: '/react/storybook?path=/story/forms-choiceinputfield--checkbox-input-field'
+---
+
+<!--
+TODO: add `validationStatus` examples when we have the validation styling done
+-->
+
+import {ComponentChecklist} from '../src/component-checklist'
+import {ChoiceInputField, Checkbox, Radio} from '@primer/react'
+import {MarkGithubIcon} from '@primer/octicons-react'
+
+The `ChoiceInputField` component is used to render a labelled checkbox or radio input and, optionally, associated hint text and graphic.
+
+## Anatomy
```

Comment:
Feels a bit heavy to go straight into this. Can we merge the descriptions here with the Props at the bottom, to save space and avoid repeating the headings? Anatomy feels more like design guidelines language too.

Reply:
I agree - these descriptions can just go under the Props section.